server/src/main/java/org/cloudfoundry/identity/uaa/audit/PredixAuditService.java

```diff
@@ -226,6 +226,28 @@ private AuditEventV2 constructPredixAuditEvent(AuditEvent auditEvent, String cor
                 category = AuditEnums.CategoryType.API_CALLS;
                 type = AuditEnums.EventType.SUCCESS_API_REQUEST;
                 break;
+            case TokenRevocationEvent:
```

Comment:
Let's discuss the assigned category/type/status values before merging this one. Some of them seem a bit off.

Reply:
TokenRevocation should go under "Authorization". Other ones seem correct.

Reply:
done.